apps/jobs/views.py

```diff
@@ -335,14 +340,19 @@ def leaderboard(request, challenge_phase_split_id):
     # while populating leaderboard
     challenge_obj = challenge_phase_split.challenge_phase.challenge
     challenge_hosts_emails = challenge_obj.creator.get_all_challenge_host_email()
+    # challenge_host = challenge_obj.creator.team_name
     is_challenge_phase_public = challenge_phase_split.challenge_phase.is_public
     # Exclude the submissions from challenge host team to be displayed on the leaderboard of public phases
     challenge_hosts_emails = [] if not is_challenge_phase_public else challenge_hosts_emails
 
     challenge_host_user = is_user_a_host_of_challenge(request.user, challenge_obj.pk)
 
     leaderboard_data = LeaderboardData.objects.exclude(
-        submission__created_by__email__in=challenge_hosts_emails)
+        Q(submission__created_by__email__in=challenge_hosts_emails) & Q(submission__baseline_submission=False))
+
+    host_participanting_teams = set(LeaderboardData.objects.filter(
```

Comment:
Why do we need this?

Reply:
This is required to get Multiple Baseline Submissions by the Host.
Ref: https://github.com/Cloud-CV/EvalAI/pull/2076/files/106a8bae10d7f2c53840b29f40a127350f929c30#diff-42c752186f5f6b2ed96b92afa242a6f3R380